sockeye/output_handler.py

```diff
@@ -48,11 +50,15 @@ class OutputHandler:
     Abstract output handler interface
     """
 
-    def handle(self, t_input: sockeye.inference.TranslatorInput, t_output: sockeye.inference.TranslatorOutput):
+    def handle(self,
```

Comment:
in order to keep this interface simple we added the struct-like objects TranslatorInput & TranslatorOutput. Would it make sense to add a walltime to TranslatourOutput to keep the outputhandler's method signatures clean?